src/dataflow/src/source/timestamp.rs

```diff
@@ -887,16 +919,6 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn source_timestamp_decoding_error() -> Result<(), String> {
```

Comment:
I've removed this test because persist is (or will be once we add the batch checksums) responsible for faithfully roundtripping the bytes, so the test didn't really seem relevant anymore. Plus now the implementation is just doing a proto decode